interop_client/src/main.rs

```diff
@@ -1477,7 +1509,7 @@ impl MlsClient for MlsClientImpl {
 
 #[derive(Parser)]
 struct Opts {
-    #[clap(long, default_value = "[::1]")]
+    #[clap(long, default_value = "0.0.0.0")]
```

Comment:
Why did you change the default to v4?

Reply:
Oh well, docker... But since docker isn't the main entry point here this should rather be solved in docker.

Reply:
While I agree, this makes it unusable on Mac and Windows...

Reply:
How is this related to the OS? I can run this without problems on Mac.

Reply:
Inside docker? I had issues with it...